src/widgets/calendar.rs

```diff
@@ -181,24 +237,86 @@ impl<DS: DateStyler> Monthly<'_, DS> {
             y += 1;
         }
     }
+    #[cfg(feature = "widget-calendar-chrono")]
+    fn render_monthly(&self, area: Rect, buf: &mut Buffer) {
```

Comment:
Seems like a bunch of duplicated code. Should either use more specific `cfg(feature` or duplicate them completely (which would also allow for `--all-features` again).